### DISCUSSION TITLE
feature request: Get function for SellerTradeParty name

### DISCUSSION THREAD

**Discussion by GorillaEXP** [ZUGFeRD/mustangproject#540] [Ideas]
Hello,

i just started to learn how to use this project and used the example from the website:

https://www.mustangproject.org/use/
`ZUGFeRDImporter zu=new ZUGFeRDImporter("MustangGnuaccountingBeispielRE-20190610_507.pdf"); 
System.out.println("Total amount: "+zu.getAmount());    `

It works fine so i tried to get more data elements from the invoice and most work, but i could not find a get-method for the name of the SellerTradeParty that is written in the xml of the invoide like this:
`<ram:SellerTradeParty>
				<ram:ID>some_id</ram:ID>
				<ram:Name>some_company_name</ram:Name>`

Is the function missing or am i doing it wrong?

**Comment by jstaerk** [comment]
Hello,
The ZUGFeRDImporter was just a quick and easy parser from the beginning which has been kept for backward compatibility, what you want to do may be possible using the ZUGFeRD**invoice**Importer where you should be able to do something like invoice.getSeller().getID().

Note besides: The invoiceimporter used to be based on the zugferdimporter. In 2.15.0 we will turn that around and the zugferdimporter will be based on the invoiceimporter. Reason: The invoiceimporter can parse both CII and UBL, zugferdimporter was only implemented in CII: if the zugferdimporter becomes just a quick way to access the invoiceimporter we only need to maintain reading anything (from both CII and UBL) in one place.


kind regards
Jochen

**Reply by GorillaEXP** [reply]
thank you for the detailled answer, this helped me!
